src/regex.rs

```diff
@@ -10,6 +12,20 @@ crate::binding_utils::gen_proto!(Regex, __str__);
 
 #[pymethods]
 impl Regex {
+    #[classmethod]
+    fn from_file(_cls: &PyType, pattern_file_content: &str) -> PyResult<Self> {
```

Comment:
```suggestion
    fn from_file(_cls: &PyType, Path: &Path) -> PyResult<Self> {
```

Reply:
`std::path` cannot be used by `pyo3` because it does not implements `PyClass` trait.

Reply:
see https://github.com/Scille/parsec-cloud/pull/3737#discussion_r1044242818